src/aco.rs

```diff
@@ -49,6 +49,7 @@ use rand::Rng;
 pub use ant_system_cfg::AntSystemCfg;
 pub use ants_system_v2::AntSystem;
 pub use ants_system_v2::probe;
+pub use ants_system_v2::builder;
```

Comment:
```suggestion
pub use ants_system::builder;
```

We should rename `ants_system_v2` module to just `ant_system` - `v2` suffix is redundant and inconsistent. However it should be done in different PR not to mix things up.